app/models/feed.rb

```diff
@@ -43,6 +44,7 @@ class Feed < ApplicationRecord
   before_validation :sanitize_text
 
   scope :default, -> { where(slug: nil) }
+  scope :custom, -> { where.not(slug: nil) }
```

Comment:
any other attribute signifiers for non-default feeds? or is it just opposite of default?

Reply:
Just the opposite of default.  Each podcast must have exactly 1 default feed (where the slug is nil).